### PR TITLE
Added Jupyter Chapter 1.0 (Setup Guide).

### PR DESCRIPTION
**Pull request by jachiang**
Has some overlap with readme.

### CODE REVIEW

File: 1.0-Workshop-Setup.ipynb [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    "## Part 4. Building bitcoind and importing the taproot library\n",
+    "\n",
+    "Please clone a fresh copy of the optech-taproot Bitcoin Core branch to your local machine.\n",
+    "* [optech-taproot Bitcoin Core branch](https://github.com/bitcoinops/bitcoin/tree/optech-taproot)\n",
+    "\n",
+    "Build the bitcoind binary for your platform according to the instructions below.\n",
+    "\n",
+    "* [Bitcoin Core build notes](https://github.com/bitcoin/bitcoin/tree/master/doc#building)"
+   ]
+  },
```
[the remark this answers is not in the document]
Comment (jachiang) [reply 1.0-Workshop-Setup.ipynb]: This is being consolidated by Schmidty [here](https://github.com/bitcoinops/taproot-workshop/pull/7).

File: 1.0-Workshop-Setup.ipynb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    "We can now continue to read this guide directly in Jupyter Notebook. \n",
+    "\n",
+    "* Git clone `optech/taproot-workshop`.  \n",
+    "* Run `jupyter notebook` in the same directory. \n",
+    "\n",
+    "You should now be able to open this interactive setup guide in Jupyter. In the next part, we will compile bitcoind with taproot support and import the Python Schnorr & Taproot libraries we need for this workshop.\n",
+    "\n",
+    "**Overview of required repositories**\n",
+    "\n",
+    "![test](images/setup0.jpg)"
```
[the remark this answers is not in the document]
Comment (jachiang) [reply 1.0-Workshop-Setup.ipynb]: [Corrected.](https://user-images.githubusercontent.com/26572234/64020330-de458200-cb20-11e9-99a4-93b5b89cdd6c.jpg) @bitschmidty, do you want to use this for the consolidated [setup guide](https://github.com/bitcoinops/taproot-workshop/pull/7)?

File: 1.0-Workshop-Setup.ipynb [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing the Python library\n",
+    "\n",
+    "Add the full path to the test framework folder of your optech-taproot bitcoin repository to the PATH environment variable. \n",
+    "\n",
+    "It should look something like this:\n",
+    "\n",
+    "* `/.../bitcoin/test/functional`"
```
[the remark this answers is not in the document]
Comment (jachiang) [reply 1.0-Workshop-Setup.ipynb]: Moving this to setup.py ...

File: 1.0-Workshop-Setup.ipynb [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "# Make sure your insert your full path.\n",
+    "sys.path.insert(0, '/.../bitcoinops/bitcoin/test/functional') \n"
```
[the remark this answers is not in the document]
Comment (jachiang) [reply 1.0-Workshop-Setup.ipynb]: This has now been incorporated in [0.1 setup chapter](https://github.com/bitcoinops/taproot-workshop/pull/7) and [1.1 Schnorr-Intro](https://github.com/bitcoinops/taproot-workshop/pull/2).

File: 1.0-Workshop-Setup.ipynb [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    "\n",
+    "* `test = TestWrapper()`\n",
+    "* `test.setup()`\n",
+    "* `test.shutdown()`\n",
+    "\n",
+    "When setup is called, the test wrapper will automatically startup a standard of 3 bitcoind daemon subprocesses, which can be accessed through the included rpc interface. We will use this frequently to broadcast Schnorr & Taproot transactions in this workshop.\n",
+    "\n",
+    "**Things to keep in mind:**\n",
+    "* Separate test wrapper setup, shutdown and your code into different cells. This ensures that an error in your code does not affect the test wrapper object and prevents orphaned subprocesses.\n",
+    "* Only run one test wrapper object at a time.\n",
```
[the remark this answers is not in the document]
Comment (jachiang) [reply 1.0-Workshop-Setup.ipynb]: This is being addressed in TestWrapper [here](https://github.com/bitcoinops/taproot-workshop/pull/8#issue-312776872), which I propose to combine with setup.py into a util.py file.

File: 1.0-Workshop-Setup.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jnewbery) [reply 1.0-Workshop-Setup.ipynb]: It doesn't look like this has been committed to the branch

**Comment by jnewbery** [comment]
Superseded by #8

**Pull request closed without merging**